packages/world/src/systems/BedSystem.sol

```diff
@@ -32,18 +33,17 @@ import { EntityId } from "../EntityId.sol";
 
 // To avoid reaching bytecode size limit
 library BedLib {
-  function transferInventory(EntityId playerEntityId, EntityId bedEntityId, ObjectTypeId objectTypeId) public {
-    transferAllInventoryEntities(playerEntityId, bedEntityId, objectTypeId);
-  }
+  function transferInventory(EntityId playerEntityId, EntityId bedEntityId, ObjectTypeId objectTypeId) public {}
```

Comment:
why is there an empty function here?

Reply:
Ah damn I missed that, we need to add a test for transferring inventory

Reply:
if you'd like to add in a separate PR could add a stub with a todo so we dont forget